torchvision/datasets/mnist.py

```diff
@@ -435,6 +435,74 @@ def get_int(b: bytes) -> int:
     return int(codecs.encode(b, 'hex'), 16)
 
 
+class MovingMNIST(VisionDataset):
```

Comment:
Can't we inherit it from `MNIST` ?

Reply:
If we use this as a video dataset, we shouldn't. We probably need to use `MNIST` to generate the training split though.